Exiled.Events/Patches/Events/Map/AnnouncingNtfEntrance.cs

```diff
@@ -84,28 +84,28 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
 
                 // if(!ev.IsAllowed)
                 //     return;
-                new(OpCodes.Call, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.IsAllowed))),
+                new(OpCodes.Callvirt, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.IsAllowed))),
                 new(OpCodes.Brfalse_S, ret),
 
                 // unitName = $"{ev.UnitName}-{ev.UnitNumber};
+                new(OpCodes.Ldarg_0),
                 new(OpCodes.Ldstr, "{0}-{1}"),
                 new(OpCodes.Ldloc_S, ev.LocalIndex),
-                new(OpCodes.Call, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.UnitName))),
+                new(OpCodes.Callvirt, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.UnitName))),
                 new(OpCodes.Ldloc_S, ev.LocalIndex),
-                new(OpCodes.Call, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.UnitNumber))),
+                new(OpCodes.Callvirt, PropertyGetter(typeof(AnnouncingNtfEntranceEventArgs), nameof(AnnouncingNtfEntranceEventArgs.UnitNumber))),
                 new(OpCodes.Box, typeof(int)),
                 new(OpCodes.Call, Method(typeof(string), nameof(string.Format), new[] { typeof(string), typeof(object), typeof(object) })),
+                new(OpCodes.Dup),
                 new(OpCodes.Starg_S, 1),
 
                 // cassieUnitName = this.GetCassieUnitName(unitName);
-                new(OpCodes.Ldarg_0),
-                new(OpCodes.Ldarg_1),
-                new(OpCodes.Call, Method(typeof(NineTailedFoxNamingRule), nameof(NineTailedFoxNamingRule.GetCassieUnitName))),
+                new(OpCodes.Callvirt, Method(typeof(NineTailedFoxNamingRule), nameof(NineTailedFoxNamingRule.GetCassieUnitName))),
                 new(OpCodes.Stloc_S, 0),
```

Comment:
Meh.

Reply:
Line 91, `Ldarg_0`
And a little upper i use `Dup` Instead of `Ldarg_1`

Reply:
Because `Stloc.S, 0` is a `cassieUnitName`, and this variable should change

Reply:
[`OpCodes.Stloc.0`](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.opcodes.stloc_0?view=net-6.0) exists.

Reply:
Oh, i'm cringe, fix that later

Reply:
Well, later means tomorrow :(
Also thank you Nao for pointing out my mistakes, i'll try to write my next transpiler without any IL viewers!

Reply:
👍🏼